### PR TITLE
Bump to Splunk 8.2.4

### PR DESCRIPTION
**Pull request by Jalkar**

### CODE REVIEW

File: roles/splunk/defaults/main.yml [hunk 1]
```diff
@@ -7,8 +7,8 @@ slack_token: undefined
 splunk_home: auto_determined # This gets set by main.yml but we have to define it here or Ansible will complain that it is undefined
 splunk_package_url: auto_determined # This gets set by main.yml but we have to define it here or Ansible will complain that it is undefined
 splunk_package_path: ~/
-splunk_package_url_full: https://d7wz6hmoaavd0.cloudfront.net/products/splunk/releases/8.2.2/linux/splunk-8.2.2-87344edfcdb4-Linux-x86_64.tgz
-splunk_package_url_uf: https://d7wz6hmoaavd0.cloudfront.net/products/universalforwarder/releases/8.2.2/linux/splunkforwarder-8.2.2-87344edfcdb4-Linux-x86_64.tgz
+splunk_package_url_full: https://download.splunk.com/products/splunk/releases/8.2.4/linux/splunk-8.2.4-87e2dda940d1-Linux-x86_64.tgz
```
Comment (dtwersky) [inline roles/splunk/defaults/main.yml:10]: We can jump to 8.2.5 already
Full: https://download.splunk.com/products/splunk/releases/8.2.5/linux/splunk-8.2.5-77015bc7a462-Linux-x86_64.tgz
UF: https://download.splunk.com/products/universalforwarder/releases/8.2.5/linux/splunkforwarder-8.2.5-77015bc7a462-Linux-x86_64.tgz

Comment (Jalkar) [reply roles/splunk/defaults/main.yml:10]: The maintainer of the project left Splunk, and Splunk do not intend to take ownership of the project... those playbook are not maintained anymore

Comment (dtwersky) [reply roles/splunk/defaults/main.yml:10]: I am currently the maintainer of this project

Comment (Jalkar) [reply roles/splunk/defaults/main.yml:10]: That's a really good news :)

**Pull request closed without merging**